### PR TITLE
Translate tsconfig `files` section into Japanese

### PR DESCRIPTION
**Pull request by Quramy**
It's related to #100 , #220 

## What I did
Translated some resources in tsconfig references into Japanese. This PR covers http://www.typescriptlang.org/v2/en/tsconfig#Project_Files_0

[Image from Gyazo](https://gyazo.com/1058b5a771397637ce42aa31b66fac53)

### CODE REVIEW

**Comment by Quramy** [comment]
I think this PR needs some Japanese reviewers.  @Natualclar , @sasurau4 would you check my translation ?

File: packages/tsconfig-reference/copy/ja/options/extends.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+display: "Extends"
+oneline: "Inherit options for a TSConfig"
+---
+
+`extends`の値は、別の継承対象の設定ファイルへのパスを含む文字列です。
+Node.jsにおけるモジュール解決の流儀が用いられます。
+
+ベースとなるファイルからの設定が最初に読み込まれ、続いて継承ファイルの設定によってオーバーライドされます。設定ファイル内のすべての相対パスは、元の設定ファイルを起点として解決されます。
+
+継承した設定ファイルの`files`, `include`および`exclude`はベースとなる設定ファイルの内容を_上書き_します。
```
Comment (Naturalclar) [inline packages/tsconfig-reference/copy/ja/options/extends.md]: [imo] For japanese text, comma-separated text should use Japanese comma.
```suggestion
継承した設定ファイルの`files`、`include`および`exclude`はベースとなる設定ファイルの内容を_上書き_します。
```

File: packages/tsconfig-reference/copy/ja/options/include.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+└── yarn.lock
+```
+
+`include`と`exclude`はグロブパターンのためのワイルドカードをサポートしています:
+
+- `*` ゼロ個以上の文字列にマッチ（ディレクトリセパレータは除く）
+- `?` 任意の1文字にマッチ（ディレクトリセパレータは除く）
+- `**/` 任意階層の任意ディレクトリにマッチ
+
+グロブパターンがファイルの拡張子を含まない場合、サポートされる拡張子のみが含まれるようになります（例：`.ts`, `.tsx`と`.d.ts`はデフォルトでインクルードされ、`.js`と`.jsx`は`allowJs`が設定された場合のみインクルードされます）。
```
Comment (Naturalclar) [inline packages/tsconfig-reference/copy/ja/options/include.md]:
```suggestion
グロブパターンがファイルの拡張子を含まない場合、サポートされる拡張子のみが含まれるようになります（例：`.ts`、`.tsx`と`.d.ts`はデフォルトでインクルードされ、`.js`と`.jsx`は`allowJs`が設定された場合のみインクルードされます）。
```

File: packages/tsconfig-reference/copy/ja/options/typeAcquisition.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+```json
+{
+  "typeAcquisition": {
+    "include": ["jest"]
+  }
+}
+```
+
+モジュールが自動で取得されるべきでない場合。例えば、そのライブラリが`node_modules`に含まれてはいるが、チームでこのライブラリを利用しないことを合意している場合：
```
Comment (Naturalclar) [inline packages/tsconfig-reference/copy/ja/options/typeAcquisition.md]: For consistency, we should keep colons to be the same byte size.
```suggestion
モジュールが自動で取得されるべきでない場合。例えば、そのライブラリが`node_modules`に含まれてはいるが、チームでこのライブラリを利用しないことを合意している場合:
```

Comment (Quramy) [reply packages/tsconfig-reference/copy/ja/options/typeAcquisition.md]: Thx. I'll use ascii `:`

**Comment by orta** [comment]
Looking good 👍

**Pull request merged**